judgels-frontends/raphael/src/modules/api/jophiel/user.tsx

```diff
@@ -9,8 +9,12 @@ export interface User {
   username: string;
 }
 
+export interface Jid {
+  jid: string;
+}
+
 export interface UsernamesMap {
-  [username: string]: User;
+  [username: string]: Jid;
```

Comment:
No, the value type is `string`, not `Jid` (`{ jid: string }`); they are different things